### PR TITLE
Fix for in author name

### PR DESCRIPTION
**Pull request by pkdash**
### Pull Request Checklist: 
- [ ] Positive Test Case Written by Dev

- [ ] Automated Testing

- [ ] Sufficient User and Developer Documentation

- [ ] Passing Jenkins Build

- [ ] Peer Code review and approval

### Positive Test Case
1. [Enter positive test case here]

### CODE REVIEW

**Comment by hydrocheck** [comment]
| METRIC | VALUE |
|:-----:|:-----:|
| NEW SECURITY HOTSPOTS | 0 |
| NEW BUGS | 0 |
| NEW DUPLICATED LINES DENSITY | 0.0 |
| NEW VULNERABILITIES | 0 |
| NEW CODE SMELLS | 0 |

[https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-4327](https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-4327)

File: hs_core/templates/pages/genericresource.html [hunk 1]
```diff
@@ -1005,7 +1005,7 @@ <h3>Resource Specific</h3>
           {% for author in creators %}
               {
                   "id": "{{ author.id }}",
-                  "name": "{{ author.name|default:"" }}",
+                  "name": "{{ author.name|default:""|escapejs }}",
```
Comment (sblack-usu) [inline hs_core/templates/pages/genericresource.html:1008]: Do we risk the same problem with organization?  Any reason not to use escapejs with organization?

Comment (sblack-usu) [reply hs_core/templates/pages/genericresource.html:1008]: What about best_name in the last_changed_by block below?

Comment (pkdash) [reply hs_core/templates/pages/genericresource.html:1008]: Yeah need to fix those too.

Comment (pkdash) [reply hs_core/templates/pages/genericresource.html:1008]: @sblack-usu Made the suggested changes.

**Review by sblack-usu — APPROVED** [review]
Looks good

**Pull request merged**